src/02_spec/07_modules/cdm/dbgregisters.rst

```diff
@@ -0,0 +1,373 @@
+Programmer Interface: Control Registers
+---------------------------------------
+
+The Control Debug Module implements the :ref:`sec:spec:api:base_register_map`.
+The reset values are to be decided.
+
+.. tabularcolumns:: |p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.40\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|
+.. flat-table:: CDM-OR1K base register reset values
+  :widths: 2 2 4 2
+  :header-rows: 1
+
+  * - address
+    - name
+    - description
+    - reset value
+
+  * - 0x0000
+    - ``MOD_VENDOR``
+    - module vendor
+    - 0x0001
+
+  * - 0x0001
+    - ``MOD_TYPE``
+    - module type identifier
+    - 0x0006
+
+  * - 0x0002
+    - ``MOD_VERSION``
+    - module version
+    - 0x0000
+
+  * - 0x0003
+    - ``MOD_CS``
+    - module control and status
+    - 0x0000
+
+  * - 0x0004
+    - ``MOD_EVENT_DEST``
+    - destination of debug events
+    - 0x0000
+
+Additionally, it implements the following debug control registers of or1k CPU core for CDM-specific functionality.
```

Comment:
I see what you're trying to express, but that's not a sentence.

How about: "Additionally, the CDM-OR1K module implements the following registers. Registers in the range xxx-yyy are forwarded to corresponding registers of the attached or1k CPU core."